examples/oneview_storage_volume_attachment.yml

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+---
+- hosts: all
+  vars:
+    config: "{{ playbook_dir }}/oneview_config.json"
+    server_profile_uri: "/rest/server-profiles/e6516410-c873-4644-ab93-d26dba6ccf0d"
+  tasks:
+   - name: Removes extra presentations from a specified server profile uri
+     oneview_storage_volume_attachment:
+       config: "{{ config }}"
+       state: "extra_presentations_removed"
+       server_profile: "{{ server_profile_uri }}"
```

Comment:
Just a comment: I would prefer see the name of the Server Profile instead of the URI, since it is supported for this attribute and improves the readability.